functions/Slack/main.py

```diff
@@ -27,8 +29,10 @@ def handle(event, context):
     slack_channel = os.environ['COFFEE_BUTTON_SLACK_CHANNEL']
 
     if event['clickType'] == str(ButtonClickType.Single):
+        now = datetime.datetime.now(tz=pytz.timezone('US/Eastern'))
+        ready_time = now + datetime.timedelta(minutes=5)
         requests.post(slack_webhook_url, json={
-            'text': 'Coffee is brewing and will be ready in about 5 minutes!',
+            'text': 'Coffee is brewing and will be ready at {}!'.format(ready_time.strftime('%-I:%M%p')),  # NOQA
```

Comment:
I was, but I was testing within the context of a Linux container (the image definition included here). Should be Debian Jessie. Unclear what's going on.